changelog.d/1616.bugfix

```diff
@@ -0,0 +1,4 @@
+Breaking change: Raises the required NodeJS version to 16.
+Fixes RoomConfig.getRoomState() returning Promise<undefined> instead of Promise<null> on timeouts. This violated the function's type signature.
```

Comment:
I wouldn't bother listing internal changes here, unless it affects the admins of the bridge in a way they can understand.